app/controllers/coffee_controller.rb

```diff
@@ -1,11 +1,14 @@
 class CoffeeController < ApplicationController
   
   def index
-    @most_recent_brew = Brew.last
+    if current_user 
+      @most_recent_brew = Brew.where(team_id: current_user.team_id).last
```

Comment:
what you could do instead is `current_user.team.brews.last` or something like that. The way you have it is fine but maybe you can play with it a bit.